pkg/backuphelpers/backupvars.go

```diff
@@ -0,0 +1,47 @@
+package backuphelpers
+
+import (
+	"fmt"
+	"strings"
+	"sync"
+
+	backupv1alpha1 "github.com/openshift/api/config/v1alpha1"
+)
+
+type BackupVar interface {
+	ArgString() string
+}
+
+type BackupConfig struct {
+	enabled  bool
+	backupCR backupv1alpha1.EtcdBackupSpec
+	mux      sync.Mutex
+}
+
+func NewDisabledBackupConfig(backupCR backupv1alpha1.EtcdBackupSpec, enabled bool) *BackupConfig {
+	return &BackupConfig{
+		enabled:  enabled,
+		backupCR: backupCR,
+		mux:      sync.Mutex{},
+	}
+}
+
+func (b *BackupConfig) ArgString() string {
+	b.mux.Lock()
+	defer b.mux.Unlock()
+
+	args := []string{"    - backup-server"}
+	if b.enabled {
+		args = append(args, fmt.Sprintf("- --%s=%v", "enabled", b.enabled))
+	}
+
+	if b.backupCR.TimeZone != "" {
+		args = append(args, fmt.Sprintf("- --%s=%s", "timezone", b.backupCR.TimeZone))
+	}
+
+	if b.backupCR.Schedule != "" {
+		args = append(args, fmt.Sprintf("- --%s=%s", "schedule", b.backupCR.Schedule))
+	}
```

Comment:
we can change that later when we fill the command with logic, but there is some logical flaw. 
If it's disabled, we shouldn't set any of those fields. If they are partially empty, we need to consider the CRD as invalid or impute proper defaults

Reply:
i think defaults will do 👍🏽

Reply:
I think the defaults will bite your a$$ in the next PR ;)